File: Algos-DataStructure/Java/Array-Hash/Pairs.java
    static int pairs(int k, int[] arr) {
        Map<Integer, Integer> pair = new HashMap<>();
        int count = 0;
        
        for (int num : arr) {
            pair.put(num, num);
        }
        
        for (int num : arr) {
            int target = pair.get(num) - k;
            if (pair.containsKey(target)) {
                count++;
            }
        }
        
        return count;
    }

File: Algos-DataStructure/JavaScript/Array-Hash/Pairs.js
function pairs(k, arr) {
    // Write your code here
    let pair = new Map();
    var count = 0;
    arr.forEach(num => {
        pair.set(num, num); 
    });
    
    arr.forEach(num =>{
        let target = pair.get(num) - k;
        if(pair.has(target)){
            count++;
        }
    })
    return count;

}
// Add each into a pair and then create another arr to see if it has target and increase count